pkg/constant/configuration.go

```diff
@@ -0,0 +1,15 @@
+package constant
```

Comment:
minor - I see the packaging and location of the constant little odd. Should we move this to controller ? Or was there any reason to introduce it here :) ?